pelita/game.py

```diff
@@ -678,16 +678,20 @@ def play_turn(game_state, allow_exceptions=False):
     if game_state['gameover']:
         raise ValueError("Game is already over!")
 
-    game_state.update(update_food_lifetimes(game_state, NOISE_RADIUS))
-    game_state.update(relocate_expired_food(game_state))
-
     # Now update the round counter
     game_state.update(next_round_turn(game_state))
 
     turn = game_state['turn']
     round = game_state['round']
     team = turn % 2
 
+    if turn >= 2:
```

Comment:
I don’t understand why this is needed. The code should remove only one lifetime unit per turn. I would prefer having it either completely symmetric (in every turn and with lifetime x 4) or perhaps *after* turn == 3 and then do it for both teams at the same time.